Formula/t/terraform.rb

```diff
@@ -36,6 +40,16 @@ def install
     system "go", "build", *std_go_args(ldflags: "-s -w")
   end
 
+  def caveats
+    <<~EOS
+      We will not accept any new Terraform releases in homebrew/core.
```

Comment:
I think it would only impact 1.6.0, right? if that is the case, we can still accept the new releases with 1.5.x